sdk/maps/azure-maps-search/tests/test_client_unit.py

```diff
@@ -0,0 +1,214 @@
+import sys
+import pytest
+try:
+    from unittest.mock import Mock
+except ImportError:  # python < 3.3
+    from mock import Mock  # type: ignore
+
+from devtools_testutils import AzureTestCase
+from azure.core.pipeline.transport import HttpTransport, HttpResponse
+from azure.core.exceptions import HttpResponseError
+from azure.core.pipeline.policies import AzureKeyCredentialPolicy
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.search import MapsSearchClient
+from azure.maps.search.models import StructuredAddress
+
+
+# cSpell:disable
+class MockTransport(HttpTransport):
```

Comment:
Why do you need tests with MockTransport when you have actual recorded tests?

Reply:
Will remove and update the unit test